### PR TITLE
Fix flaky ProfileWatcherTest.kt

### PR DESCRIPTION
**Pull request by hunterwerlla**
- On my machine this test would fail about 1/100 runs, but it has been failing a lot more often on CI recently.

## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts [hunk 1]
```diff
@@ -131,7 +131,7 @@ afterEvaluate {
 
         systemProperty("aws.telemetry.skip_prompt", "true")
         systemProperty("aws.suppress_deprecation_prompt", true)
-        ciOnly {
+        ciOnly() {
```
Comment (rli) [inline buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts:134]: why this change?

**Review by rli — APPROVED** [review]

File: buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts (hunk 1 above)
Comment (hunterwerlla) [reply buildSrc/src/main/kotlin/toolkit-intellij-subplugin.gradle.kts:134]: IDE suggested it and I had it open, and I was thinking there was an issue with it (there wasn't), also too small to warrant its own PR IMO
